src/main/java/it/pagopa/swclient/mil/dao/TerminalEntity.java

```diff
@@ -2,18 +2,28 @@
 
 import io.quarkus.mongodb.panache.common.MongoEntity;
 import org.bson.codecs.pojo.annotations.BsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.bson.codecs.pojo.annotations.BsonId;
 
+@ToString
+@EqualsAndHashCode
+@Getter
+@Setter
 @MongoEntity(database = "mil", collection = "terminalRegistry")
 public class TerminalEntity {
 
+    @BsonId
```

Comment:
We have this error on the service during the postman test execution:

`2024-04-19T12:51:47.101947670Z Caused by: org.bson.codecs.configuration.CodecConfigurationException: Failed to decode 'TerminalEntity'. Decoding 'terminalUuid' errored with: readString can only be called when CurrentBSONType is STRING, not when CurrentBSONType is OBJECT_ID.
2024-04-19T12:51:47.101951158Z  at org.bson.codecs.pojo.PojoCodecImpl.setPropertyValue(PojoCodecImpl.java:236)
2024-04-19T12:51:47.101954269Z  at org.bson.codecs.pojo.PojoCodecImpl.decodePropertyModel(PojoCodecImpl.java:202)
2024-04-19T12:51:47.101972030Z  at org.bson.codecs.pojo.PojoCodecImpl.decodeProperties(PojoCodecImpl.java:191)
2024-04-19T12:51:47.101975862Z  at org.bson.codecs.pojo.PojoCodecImpl.decode(PojoCodecImpl.java:100)
2024-04-19T12:51:47.101979166Z  at org.bson.codecs.pojo.PojoCodecImpl.decode(PojoCodecImpl.java:104)
2024-04-19T12:51:47.101982786Z  at org.bson.codecs.pojo.AutomaticPojoCodec.decode(AutomaticPojoCodec.java:37)
2024-04-19T12:51:47.101985965Z  ... 27 more
2024-04-19T12:51:47.101990751Z Caused by: org.bson.BsonInvalidOperationException: readString can only be called when CurrentBSONType is STRING, not when CurrentBSONType is OBJECT_ID.
2024-04-19T12:51:47.101994118Z  at org.bson.AbstractBsonReader.verifyBSONType(AbstractBsonReader.java:689)
2024-04-19T12:51:47.101997462Z  at org.bson.AbstractBsonReader.checkPreconditions(AbstractBsonReader.java:721)
2024-04-19T12:51:47.102001021Z  at org.bson.AbstractBsonReader.readString(AbstractBsonReader.java:456)
2024-04-19T12:51:47.102004729Z  at org.bson.codecs.StringCodec.decode(StringCodec.java:80)
2024-04-19T12:51:47.102007937Z  at org.bson.codecs.StringCodec.decode(StringCodec.java:31)
2024-04-19T12:51:47.102011333Z  at org.bson.codecs.pojo.LazyPropertyModelCodec.decode(LazyPropertyModelCodec.java:52)
2024-04-19T12:51:47.102014626Z  at org.bson.codecs.DecoderContext.decodeWithChildContext(DecoderContext.java:96)
2024-04-19T12:51:47.102018041Z  at org.bson.codecs.pojo.PojoCodecImpl.lambda$decodePropertyModel$2(PojoCodecImpl.java:212)
2024-04-19T12:51:47.102022880Z  at org.bson.codecs.pojo.PojoCodecImpl.setPropertyValue(PojoCodecImpl.java:234)
2024-04-19T12:51:47.102026266Z  ... 32 more
2024-04-19T12:51:47.102029697Z }`


I don't know which is the correct way to fix it, maybe we can try this  https://stackoverflow.com/questions/51888126/readstring-can-only-be-called-when-currentbsontype-is-string-not-when-currentbs